tests/Assertions/BodyMatchesTest.php

```diff
@@ -25,7 +25,7 @@ public function itFailsIfTheExpectedBodyDoesNotMatchTheActualBody()
         );
 
         $this->expectException(ExpectationFailedException::class);
-        $expectation($actual, new Muzzle);
```

Comment:
The changes in this file avoid a warning since the expectation method only expects one argument to be passed.